docs/Reference/Rest_API/Rest.md

```diff
@@ -68,16 +68,16 @@ The [validator client API methods](https://ethereum.github.io/keymanager-APIs/)
 Enable the validator client API service from the command line by setting the
 [`--validator-api-enabled`](../CLI/CLI-Syntax.md#validator-api-enabled) command line option to `true`.
 
-You can also interact with APIs by enabling the API documentation endpoint by setting the
-[`--validator-api-docs-enabled`](../CLI/CLI-Syntax.md#validator-api-docs-enabled) option to `true`.
+Alternatively, the [OpenAPI specifications](https://swagger.io/specification/) for the validator client API can be consumed by an interface such as [swagger-docs](https://swagger.io/docs/).
+In order to generate the specifications, you must first enable the API documentation endpoint.
```

Comment:
The [OpenAPI specifications](https://swagger.io/specification/) for the validator client API are available at `/swagger-docs` when `--validator-api-docs-enabled` is `true`.
The `/swagger-docs` endpoint will define the API if code generators are in use.

Basically removed the use of 'alternatively', Im not sure it fits well, and tried to better explain why you'd enable the endpoint